docs/FAQ.md

```diff
@@ -515,6 +515,27 @@ by `jj rebase` to rebase the changes in a commit. It's used in `jj log` to
 indicate which commits are empty. It's used in the `files()` revset function
 (and by `jj log <path>`) to find commits that modify a certain path. And so on.
 
+## How do I revert a merge commit? `jj revert <merge>` does nothing
+
+Instead use `jj restore --from <first parent>` to revert the changes merged
+from the second parent.
+
+Example:
+```text
+@
+|
+C
+| \
+B D
+|/
+A
+```
+To revert the merge in `C`, `jj restore --from B` and then describe the message
+with something like `jj desc -m "Revert merge <C's commit-id>`. As this removes
```

Comment:
I'd add "with" after "Revert merge".

Reply:
It's a little arguable whether "Revert merge with D" is clear, but I think "Revert merge C" is definitely *not* clear. So, if we don't have a good idea for a description here that makes things clearer, we could just delete the suggestion "... then describe the message with something like ...".

Reply:
I took your last suggestion.